python/mrchem/api.py

```diff
@@ -127,7 +127,8 @@ def write_molecule(user_dict, origin):
         "charge": mol.charge,
         "coords": mol.get_coords_in_program_syntax(),
     }
-    if user_dict["WaveFunction"]["environment"].lower() == "pcm":
+    
+    if (user_dict["WaveFunction"]["environment"].lower()).split("_")[0] == "pcm":
```

Comment:
wouldn't this work as well?
```suggestion
    if "pcm" in user_dict["WaveFunction"]["environment"].lower():
```

Reply:
I think I tried something similar, we can add it just to see if it still works